### ISSUE TITLE
New checkpoints

### ISSUE THREAD

**Issue by zhvng** [LAION-AI/CLAP#75]
Hey all! Appreciate the work you've done to make CLAP a pip library. I noticed the checkpoints are now hosted on HuggingFace, and was wondering whether these new checkpoints were the same as the old ones hosted [here](https://drive.google.com/drive/folders/1Ni8lZ2pryTESjgq8gELLQNM_HGdWtFrE). Is the model performance the same for the new checkpoints and code, or has the model itself been updated?

**Comment by lukewys** [comment]
Hi, here are the change in the checkpoints:
1. for the checkpoints without fusion, the model itself is not retrained. Previously, we selected the checkpoint with best retrieval performance. However, we found that such model selected is overfitted and has worse downstream performance (e.g., esc50 zeroshot performance). Thus, for the new checkpoint, we select the ones with the best ESC50 performance. The checkpoints with ESC50 performance have very little retrieval performance difference with the model that achieves the best esc50 performance.
2. for the models with fusion, we fixed a bug in fusion model (see more here: https://github.com/LAION-AI/CLAP/issues/62) and retrained the model, selecting the checkpoint using the principle described in 1#.
 
Overall, we expect the new checkpoints to perform better in various downstream tasks. However, we would be happy to know your result if you are using our model in your task and comparing the checkpoints.
